src/Microsoft.DncEng.Configuration.Extensions/LocalDevTokenCredential.cs

```diff
@@ -111,7 +111,6 @@ private static Task DeviceCodeResultCallback(DeviceCodeResult arg)
         {
             if (!Debugger.IsAttached)
                 throw new InvalidOperationException("Debugger required for local service fabric authentication.");
-
```

Comment:
This file shouldn't have any changes

Reply:
there was merge conflict, got the file from master and fixed this